tests/integ/modin/frame/test_set_index.py

```diff
@@ -80,22 +80,6 @@ def test_set_index_multiindex_columns(snow_df):
     )
 
 
-@sql_count_checker(query_count=2)
-def test_set_index_negative(snow_df, native_df):
```

Comment:
Do we have test to show current behavior when the lengths do not match?

Reply:
Added one!